src/app/parcels/parcelsTable/styles.ts

```diff
@@ -11,38 +11,42 @@ export const PreTableControls = styled.div`
 
 export const parcelTableColumnStyleOptions = {
     iconsColumn: {
-        width: "4rem",
+        maxWidth: "4rem",
+        minWidth: "0.7rem",
+        grow: 0.5,
     },
```

Comment:
Allows icons column to shrink when viewed on a smaller screen

Reply:
Is that just space around icons or icons themselves?

Reply:
Just the space around